src/main/java/no/nav/pam/annonsemottak/stilling/Stilling.java

```diff
@@ -327,16 +325,12 @@ public Stilling merge(Stilling stilling) {
         this.uuid = stilling.getUuid();
         this.setCreated(stilling.getCreated());
 
-        if(!kilde.equals(Kilde.SBL.value())) {
-            setPublished(stilling.getPublished());
-        }
-
         return this;
     }
 
     public Stilling stopIfExpired(Stilling stilling) {
 
-        if(!kilde.equals(Kilde.XML_STILLING.value())) {
+        if(!kilde.equals(Kilde.XML_STILLING.toString())) {
```

Comment:
Denne er inkonsekvent i annonsemottaket. Value og toString brukes om hverandre og burde kanskje rettes, men hører kanskje ikke hjemme i denne PRen

Reply:
Jeg tror det er riktig og bevege seg mot toString ;-)

Reply:
Det har du helt rett i, bruken er inkonsekvent og skaper forvirring. Diskuterte dette med @oyvindstegard og ble enig om å løse dette som en egen sak/PR. Det som gjør dette litt kronglete er at de to verdiene har havnet i databasen (externalrun) så det skal litt mer til enn bare en runde med enkel refakorering for å rydde opp i dette.